skl2onnx/_parse.py

```diff
@@ -559,7 +560,18 @@ def parse_sklearn(scope, model, inputs, custom_parsers=None, final_types=None):
 
         reserved = []
         for o in outputs:
-            reserved.append(scope.reserve_name(o.raw_name))
+            if scope.is_reserved(o.raw_name):
+                from logging import getLogger
+                mes = (
+                    "Name %r is reserved. It is usually due to input and "
+                    "output sharing the same name (not allowed). It will "
+                    "be renamed in %r." % (o.raw_name, o.onnx_name))
```

Comment:
```suggestion
                    "Name %r is reserved. It is usually due to input and "
                    "output sharing the same name (not allowed). It will "
                    "be renamed to %r." % (o.raw_name, o.onnx_name))
```